diff --git a/CMakeLists.txt b/CMakeLists.txt
index da898b096..a82f01ba9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -478,6 +478,8 @@ elseif (NOT TUNE_CPU STREQUAL "none")
     message(FATAL_ERROR "Trying to compile cpu type '${TUNE_CPU}' but the compiler doesn't support this")
   endif()
 endif()
+message(STATUS "Trying to compile cpu type '${TUNE_CPU}'")
+message(STATUS "Trying to compile arch type '${TUNE_ARCH}'")
 
 set(GIT_DESCRIBE_STRING "FEX-Unknown")
 
