diff --git a/src/libkrun/Cargo.toml b/src/libkrun/Cargo.toml
index 8aa7929..07d4408 100644
--- a/src/libkrun/Cargo.toml
+++ b/src/libkrun/Cargo.toml
@@ -15,7 +15,6 @@ efi = [ "blk", "net" ]
 gpu = ["krun_display"]
 snd = []
 virgl_resource_map2 = []
-nitro = [ "dep:nitro", "dep:nitro-enclaves" ]
 
 [dependencies]
 crossbeam-channel = ">=0.5.15"
@@ -38,8 +37,6 @@ hvf = { path = "../hvf" }
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = { version = ">=0.11", features = ["fam-wrappers"] }
 kvm-ioctls = ">=0.21"
-nitro = { path = "../nitro", optional = true }
-nitro-enclaves = { version = "0.3.0", optional = true }
 vm-memory = { version = ">=0.13", features = ["backend-mmap"] }
 
 [lib]
