diff --git a/src/arch/Cargo.toml b/src/arch/Cargo.toml
index 3b2dee0..9e04d85 100644
--- a/src/arch/Cargo.toml
+++ b/src/arch/Cargo.toml
@@ -23,7 +23,7 @@ utils = { package = "krun-utils", version = "=0.1.0-1.18.0", path = "../utils" }
 
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = { version = "0.13", features = ["fam-wrappers"] }
-kvm-ioctls = "0.22"
+kvm-ioctls = "0.23"
 
 [dev-dependencies]
 utils = { package = "krun-utils", version = "=0.1.0-1.18.0", path = "../utils" }
diff --git a/src/cpuid/Cargo.toml b/src/cpuid/Cargo.toml
index d0141e3..1294742 100644
--- a/src/cpuid/Cargo.toml
+++ b/src/cpuid/Cargo.toml
@@ -15,4 +15,4 @@ vmm-sys-util = "0.14"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = { version = "0.13", features = ["fam-wrappers"] }
-kvm-ioctls = "0.22"
+kvm-ioctls = "0.23"
diff --git a/src/devices/Cargo.toml b/src/devices/Cargo.toml
index 70a6350..ea0dfc6 100644
--- a/src/devices/Cargo.toml
+++ b/src/devices/Cargo.toml
@@ -51,7 +51,7 @@ hvf = { package = "krun-hvf", version = "=0.1.0-1.18.0", path = "../hvf" }
 rutabaga_gfx = { package = "krun-rutabaga-gfx", version = "=0.1.0-1.18.0", path = "../rutabaga_gfx", features = ["x"], optional = true }
 caps = "0.5.5"
 kvm-bindings = { version = "0.13", features = ["fam-wrappers"] }
-kvm-ioctls = "0.22"
+kvm-ioctls = "0.23"
 
 [target.'cfg(any(target_arch = "aarch64", target_arch = "riscv64"))'.dependencies]
 vm-fdt = ">= 0.2.0"
diff --git a/src/libkrun/Cargo.toml b/src/libkrun/Cargo.toml
index 63ab07a..a0135f2 100644
--- a/src/libkrun/Cargo.toml
+++ b/src/libkrun/Cargo.toml
@@ -41,7 +41,7 @@ hvf = { package = "krun-hvf", version = "=0.1.0-1.18.0", path = "../hvf" }
 
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = { version = "0.13", features = ["fam-wrappers"] }
-kvm-ioctls = "0.22"
+kvm-ioctls = "0.23"
 vm-memory = { version = "0.17", features = ["backend-mmap"] }
 
 [lib]
diff --git a/src/vmm/Cargo.toml b/src/vmm/Cargo.toml
index 544410a..39c3dda 100644
--- a/src/vmm/Cargo.toml
+++ b/src/vmm/Cargo.toml
@@ -44,7 +44,7 @@ zstd = "0.13"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = { version = "0.13", features = ["fam-wrappers"] }
-kvm-ioctls = "0.22"
+kvm-ioctls = "0.23"
 
 [target.'cfg(target_os = "macos")'.dependencies]
 hvf = { package = "krun-hvf", version = "=0.1.0-1.18.0", path = "../hvf" }
