diff --git a/libkrun-1.14.0/src/libkrun/Cargo.toml b/libkrun-1.14.0/src/libkrun/Cargo.toml
index 195e456..94d20bb 100644
--- a/libkrun-1.14.0/src/libkrun/Cargo.toml
+++ b/libkrun-1.14.0/src/libkrun/Cargo.toml
@@ -14,7 +14,6 @@ efi = [ "blk", "net" ]
 gpu = []
 snd = []
 virgl_resource_map2 = []
-nitro = [ "dep:nitro", "dep:nitro-enclaves" ]
 
 [dependencies]
 crossbeam-channel = ">=0.5.15"
@@ -35,8 +34,6 @@ hvf = { path = "../hvf" }
 [target.'cfg(target_os = "linux")'.dependencies]
 kvm-bindings = { version = ">=0.11", features = ["fam-wrappers"] }
 kvm-ioctls = ">=0.21"
-nitro = { path = "../nitro", optional = true }
-nitro-enclaves = { version = "0.2.0", optional = true }
 vm-memory = ">=0.13"
 
 [lib]
