diff --git i/meson.build w/meson.build
index aa485a262f5..f313aa78099 100644
--- i/meson.build
+++ w/meson.build
@@ -763,8 +763,8 @@ endif
 with_virtgpu_kumquat = get_option('virtgpu_kumquat') and with_gfxstream_vk
 if with_gallium_rusticl or with_nouveau_vk or with_tools.contains('etnaviv') or with_virtgpu_kumquat
   # rust.bindgen() does not pass `--rust-target` to bindgen until 1.7.0.
-  if meson.version().version_compare('< 1.7.0')
-    error('Mesa Rust support requires Meson 1.7.0 or newer')
+  if meson.version().version_compare('< 1.5.0')
+    error('Mesa Rust support requires Meson 1.5.0 or newer')
   endif
 
   add_languages('rust', required: true)
